File: EJERCICIO_4/EJERCICIOSJAVA/License.java
package EJERCICIOSJAVA;
public class License {
    private double price;
    private int duration;
    private String type;

    public License(double price, int duration, String type) {
        this.price = price;
        this.duration = duration;
        this.type = type;
    }

    public double getPrice() {
        return price;
    }

    public int getDuration() {
        return duration;
    }

    public String getType() {
        return type;
    }
}

File: EJERCICIO_4/EJERCICIOSPHP/License.php
<?php
class License{
    private $price;
    private $duration;
    private $type;

    public function __construct( $price, $duration, $type ){
        $this->price = $price;
        $this->duration = $duration;
        $this->type = $type;
    }

    public function getPrice(){
        return $this->price;
    }
    public function getDuration(){
        return $this->duration;
    }
    public function getType(){
        return $this->type;
    }
}
?>